performant-software/DM
# Add multi-user project dashboard alert

### What does this PR do?
- Adds a frontend alert from the data provided by the projects/dashboard API endpoint. If other users are recorded as editing a project, a new user upon opening that project will see an alert warning them of possible save conflicts and providing the names and email addresses of those other users.
- Fixes a very specific case in which auto project selection for guest users could trigger a loop
- Increases default rollover window size

### What issues does it address?
Closes #129 

### How to test
Log in to http://dm-staging.middell.net/ from a couple different accounts on different machines/browsers and open the same project. When opening a project for which an account is already signed in, you should see an alert dialog with red header text before you see the project's resource list. This dialog should contain names and email addresses for those other accounts and should have a "Continue" button to let you proceed to the resource list.
